Package.swift

```diff
@@ -4,13 +4,10 @@ import PackageDescription
 let package = Package(
     name: "vapor-template",
     dependencies: [
-        .package(url: "https://github.com/vapor/vapor.git", from: "3.3.0"),
-        .package(url: "https://github.com/vapor/fluent-mysql.git", from: "3.0.1"),
-        .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", from: "6.0.0")
+        .package(url: "https://github.com/youclap/vapor-shared.git", .branch("master"))
```

Comment:
Maybe we should do releases, instead of using directly master?

Reply:
yeap